core/src/main/kotlin/fr/sncf/osrd/utils/SimInfraAdapterComparatorUtils.kt

```diff
@@ -134,6 +136,92 @@ open class ComparableChunk(simInfra: RawInfra, chunkId: TrackChunkId) {
     }
 }
 
+open class ComparableSectionEndpoint(val sectionName: String, val endpoint: Endpoint) {
```

Comment:
I don't think we need to make them `open`

But they could be a `data class`, that way we get default `equals` and `hashCode`

Reply:
Done in https://github.com/osrd-project/osrd/pull/6799/commits/e92c7868d8f4b38110d616328fbca5167dd5e72c